src/services/subject.js

```
@@ -17,14 +17,16 @@ const subjectService = {
     return subject
   },
 
-  getSubjects: async (limit, offset, search) => {
+  getSubjects: async (limit, offset, search, categoryId) => {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)